examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL2/SDL.h>
+
+int main(int argc, char *argv){
```

Comment:
> Did it work?

I mean simple example without SDL. Literally `int main(int argc, char** argv) { std::cout << hello; }`. Just a sanity check after you said `that function signature is required for windows` ;)